apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js

```diff
@@ -124,17 +124,22 @@ const RequestAssignment = ({ issue }) => {
 
 RequestAssignment.propTypes = issueShape
 
-const HoursInput = styled(TextInput.Number)`
-  height: 32px;
+const HoursInput = styled(TextInput.Number).attrs({
```

Comment:
TextInput.Number is deprecated, please use TextInput instead
https://github.com/aragon/aragon-ui/blob/newstyle/src/components/Input/TextInput.js#L142
```suggestion
const HoursInput = styled(TextInput).attrs({
  type: 'number',
```